.config/forest.dic

```diff
@@ -19,3 +19,12 @@ RPC
 parameterize/D
 timestamp/SM
 synchronizer
+pointer/S
+validator/S
+threadsafe
```

Comment:
I believe common writing is `thread-safe` so we shouldn't have an exception here